code/game/gamemodes/cult/runes.dm

```diff
@@ -712,7 +712,7 @@ structure_check() searches for nearby cultist structures required for the invoca
 		fail_invoke()
 		log_game("Summon Cultist rune failed - target died")
 		return
-	if(cultist_to_summon.pulledby || cultist_to_summon.buckled)
+	if(cultist_to_summon.pulledby.len || cultist_to_summon.buckled)
```

Comment:
```suggestion
	if(cultist_to_summon.pulledby || cultist_to_summon.buckled)
```

Empty lists are boolean false

Reply:
Not the case im afraid. Any reference (object, datum or lost) is considered true when used in an if statement.